kinto/core/storage/postgresql/migrations/migration_011_012.sql

```diff
@@ -0,0 +1,7 @@
+-- Select all existing records and delete their tombstone if any.
+DELETE FROM deleted d
+WHERE (d.id, d.parent_id, d.collection_id) IN
+  (SELECT r.id, r.parent_id, r.collection_id FROM records r);
```

Comment:
This should work, it can also be written like that:
```sql
DELETE 
FROM deleted d 
USING records r
WHERE d.id = r.id AND d.parent_id = r.parent_id AND d.collection_id = r.collection_id
```